src/mac/MacDisplayInfo.mm

```diff
@@ -0,0 +1,60 @@
+#include "DisplayInfo.h"
+#include "UserInteractions.h"
+#include <CoreFoundation/CoreFoundation.h>
+#include <Cocoa/Cocoa.h>
+#include "vstgui/lib/platform/mac/cocoa/nsviewframe.h"
+
+namespace Surge
+{
+    namespace GUI
+    {
```

Comment:
Do we want to indent namespaces in the cpp-file at least? Wastes screen space more than brings value. I would just not indent these i.e.

```
namespace Surge
{

namespace GUI
{

NSView *getViewFromFrame(CFrame *f)
```

Reply:
Thanks. I indent headers that way but generally not source. Happy to adapt.